sql-create/create_sql_tables.sql

```diff
@@ -0,0 +1,69 @@
+-- Works table
+CREATE TABLE works (
```

Comment:
this is certainly the operation we want but this file should be removed in the place of sqlalchemy functionality.

Reply:
I agree. Please use sqlalchemy.